pkg/cluster/platformworkloadidentities.go

```diff
@@ -0,0 +1,50 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/msi/armmsi"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+func (m *manager) platformWorkloadIdentityIDs(ctx context.Context) error {
+	var err error
+	if !m.doc.OpenShiftCluster.UsesWorkloadIdentity() {
```

Comment:
Is there any particular reason we call this function here, but not in `clusterIdentityIDs`?

Reply:
I just forgot to add it there. I'll add it.